css/style.css

```diff
@@ -0,0 +1,341 @@
+  font-size: 10px;
+}
+.title-menu li:hover,
+.title-menu a:hover,
+.title-social li:hover,
+.title-social a:hover {
+  cursor: pointer;
+}
+#icon {
+  font-weight: bolder;
+  color: white;
+  margin: auto;
+  padding: 8px 10px;
+  margin-right: 20px;
+  cursor: pointer;
+}
+.header #icon a img {
+  width: 131px;
+}
+
+.dark {
+  width: 100%;
+  display: none;
+}
+.light {
+  width: 100%;
+  display: block;
+}
+#icon:hover {
+  background-color: #ddf0f9;
+}
+#icon:hover > a .light {
+  display: none;
+}
+#icon:hover > a .dark {
+  display: block;
+}
+
+/* change the color of the li elements when the mouse is on hover */
+.title-menu:hover,
+.title-social:hover {
+  background-color: #ddf0f9;
+  color: #00aeef;
+}
+
+/* puts the li elements on the social class side by side */
+.social {
+  justify-content: flex-end;
+}
+/* puts the li elements on the menu class side by side */
+.menu {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.menu-links {
+  display: flex;
+  justify-content: space-around;
+}
+
+.social-links {
+  display: flex;
+  justify-content: space-around;
+}
+
+.hamburger {
+  display: none;
+}
+
+/* HAMBURGER VIEW */
+
+.hamburger div {
+  width: 25px;
+  height: 3px;
+  margin: 5px;
+  background-color: whitesmoke;
+  transition: all 0.3s ease;
+}
+
+@media screen and (max-width: 1024px) {
+  .navigation li {
+    width: 30%;
+  }
+}
+
+@media screen and (max-width: 860px) {
+  body {
+    overflow-x: hidden;
+  }
+  .submenu-container,
+  .submenu,
+  .sub-articles-container,
+  .subArticles {
+    display: none;
+  }
+  .menu li:hover > .submenu-container,
+  .submenu,
+  .more,
+  .menu__more-articles {
+    display: none;
+    height: 0;
+    width: 0;
+    margin: 0;
+    padding: 0;
+  }
+
+  .social {
+    display: none;
+  }
+
+  .header {
+    position: inherit;
+    padding-left: 50px;
+  }
+  #icon {
+    display: block;
+    position: inherit;
+    top: 0;
+    /* bottom: 0; */
+  }
+  .menu-links {
+    position: absolute;
+    left: 0;
+    height: 98.5vh;
+    top: 3.45vh;
+    background-color: #555;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 50%;
+    transform: translateX(-100%);
+    transition: transform 0.5s ease-in;
+  }
+  .hamburger {
+    display: block;
+    cursor: pointer;
+    position: relative;
+    right: calc(60%);
+  }
+  .menu-links .title-menu:not(:first-child) {
+    opacity: 0;
+    color: white;
+    transition: transform 0.5s ease-in;
+  }
+  .menu-links .title-menu:hover .title-name {
+    color: #00aeef;
+  }
+}
+
+.nav-active {
+  transform: translateX(0%);
+}
+@keyframes navLinkFade {
+  from {
+    opacity: 0;
+    transform: translateX(-50px);
+  }
+  to {
+    opacity: 1;
+    transform: translateX(0px);
+  }
+}
+
+.toggle .line1 {
+  transform: rotate(-45deg) translate(-5px, 6px);
+}
+.toggle .line2 {
+  opacity: 0;
+}
+.toggle .line3 {
+  transform: rotate(45deg) translate(-5px, -6px);
+}
+/* HAMBURGER VIEW */
```

Comment:
Missing line below and extra line at the bottom of the comment

Reply:
👍